course/src/matrix3.cpp

```diff
@@ -90,6 +91,28 @@ double Matrix3::det() const {
   return A - B + C;
 }
 
+Matrix3 Matrix3::product(const Matrix3& r_matrix) const {
+  Matrix3 aux;
+  for (int i = 0; i < 3; ++i) {
+    for (int j = 0; j < 3; ++j) {
+      for (int iter = 0; iter < 3; ++iter) {
+        aux[i][j] += (*this)[iter][i] * r_matrix[j][iter];
+      }
+    }
+  }
+  return aux;
+}
```

Comment:
nit: Each product between `i_col` vector and `j_row` vector is the `dot` product.

Reply:
Addressed.